packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -92,23 +94,22 @@ export const RelationInput = ({
                   <Box paddingTop={1} paddingBottom={1}>
                     {href ? (
                       <BaseLink disabled={disabled} href={href}>
-                        {title}
+                        {mainField}
                       </BaseLink>
                     ) : (
-                      title
+                      mainField
                     )}
                   </Box>
 
-                  {isDraft !== undefined && (
-                    <Badge
-                      borderSize={1}
-                      borderColor={`${badgeColor}200`}
-                      backgroundColor={`${badgeColor}100`}
-                      textColor={`${badgeColor}700`}
-                    >
-                      {isDraft ? 'Draft' : 'Published'}
-                    </Badge>
-                  )}
+                  {/* TO FIX: not showing badge if D&P is not activated  */}
```

Comment:
@gu-stav I don't know where we're at for this use case but I was wondering if we could use the same logic as `Option` component:
`const hasDraftAndPublish = has(get(props, 'data'), 'isDraft');`

btw we'll we have a isDraft key if D&P is not activated? 🤔

Reply:
I haven't thought about it yet, but to be consistent I'll pass it somehow similar to `isDraft` for you, so that this component doesn't need to handle it.

Reply:
Answered below ⬇️